pom.xml

```diff
@@ -173,7 +173,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <vertx.version>3.5.4</vertx.version>
+    <vertx.version>3.9.4</vertx.version>
```

Comment:
This doesn't upgrade all vertx modules:
```
mvn dependency:tree -Dincludes=io.vertx
[INFO] org.folio:mod-inventory:jar:16.2.0-SNAPSHOT
[INFO] +- io.vertx:vertx-core:jar:3.9.4:compile
[INFO] +- io.vertx:vertx-web:jar:3.9.4:compile
[INFO] |  +- io.vertx:vertx-web-common:jar:3.9.4:compile
[INFO] |  +- io.vertx:vertx-auth-common:jar:3.9.4:compile
[INFO] |  \- io.vertx:vertx-bridge-common:jar:3.9.4:compile
[INFO] +- org.folio:mod-source-record-storage-client:jar:4.1.1:compile
[INFO] |  \- org.folio:domain-models-runtime:jar:31.0.2:compile
[INFO] |     +- io.vertx:vertx-dropwizard-metrics:jar:3.9.2:compile
[INFO] |     \- io.vertx:vertx-pg-client:jar:3.9.2:compile
[INFO] |        \- io.vertx:vertx-sql-client:jar:3.9.2:compile
[INFO] +- org.folio:mod-pubsub-client:jar:1.3.3:compile
[INFO] |  \- org.folio:folio-di-support:jar:1.1.0:compile
[INFO] |     \- io.vertx:vertx-service-proxy:jar:3.9.0:compile
[INFO] +- io.vertx:vertx-unit:jar:3.9.4:test
[INFO] \- io.vertx:vertx-codegen:jar:3.9.4:test
```
vertx-dropwizard-metrics and vertx-service-proxy have still an old version, and more vertx module may get affected whenever a dependency is added or changed in mod-inventory pom.xml.
Please consider using [Vert.x Bill of Materials (Maven BOM)](https://github.com/vert-x3/vertx-stack#bills-of-materials-maven-bom) (content: [vertx-dependencies pom.xml](https://github.com/vert-x3/vertx-dependencies/blob/3.9.4/pom.xml#L49-L54)) for io.vertx:* as suggested in the [RMB upgrading notes](https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-300).